### ISSUE TITLE
Unable to view image at admin panel!

### ISSUE THREAD

**Issue by Waheedsarooj** [SlavaYurthev/Slider-M2#6]
Hi,

I have installed magento2 under a directory, I verified that images uploaded successfully but when it come to show on pages (list and edit page) it won't work it pulling images from the root rather than include the directory in which magento installed.

Can you please advise.

Waheed

**Comment by SlavaYurthev** [comment]
Do you use "root_dir/pub" directory instead of "root_dir"?

**Comment by SlavaYurthev** [comment]
you can edit path by line:
https://github.com/SlavaYurthev/Slider-M2/blob/master/Ui/Component/Listing/Column/Image.php#L31

Will be fixed soon... (in the next update)
thanks for your report.

**Comment by Waheedsarooj** [comment]
So I need to hard code the path?

**Comment by SlavaYurthev** [comment]
You can try this:
```
$store = $this->storeManager->getStore();
$baseUrl = $store->getBaseUrl(\Magento\Framework\UrlInterface::URL_TYPE_WEB);
$item['image_src'] = rtrim($baseUrl, '/').$item['image'];
```

**Comment by Waheedsarooj** [comment]
Yes its working but only on listing page, whereas in edit mode image still not showing.

**Comment by SlavaYurthev** [comment]
https://github.com/SlavaYurthev/Slider-M2/blob/master/Block/Adminhtml/Items/Edit/Tab/General.php
```
protected $_storeManager;
public function __construct(
...
) {
    $this->_wysiwygConfig = $wysiwygConfig;
    $this->_storeManager = $context->getStoreManager();
    parent::__construct($context, $registry, $formFactory, $data);
}
```
add new function like:
```
private function getImageUrl($model){
    $store = $this->_storeManager->getStore();
    $baseUrl = $store->getBaseUrl(\Magento\Framework\UrlInterface::URL_TYPE_WEB);
    return rtrim($baseUrl, '/').'/'.ltrim($model->getData('image'), '/');
}
```
and use $this->getImageUrl($model) instead of $model->getData('image') in line 54 and 55

**Comment by SlavaYurthev** [comment]
Fix at 0.4.1
